### PR TITLE
AKS: Add validation for PremiumV2 disks and fixes in error messages

### PR DESCRIPTION
**Pull request by mukhoakash**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally? (`pip install wheel==0.30.0` required)
- [ ] My extension version conforms to the [Extension version schema](https://github.com/Azure/azure-cli/blob/release/doc/extensions/versioning_guidelines.md)

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
Thank you for your contribution! We will review the pull request and get back to you soon.

**Review by FumingZhang — COMMENTED** [review]
lgtm

Queued [live test](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=98801192&view=results), test passed!
- test_aks_create_with_azurecontainerstorage
- test_aks_create_with_azurecontainerstorage_with_ephemeral_disk_parameters
- test_aks_create_with_azurecontainerstorage_with_nodepool_name
- test_aks_update_with_azurecontainerstorage
- test_aks_update_with_azurecontainerstorage_with_ephemeral_disk_parameters

Please fix failed CI 
https://dev.azure.com/azclitools/public/_build/results?buildId=174946&view=logs&j=506b8f13-8269-5210-246e-a3964e53141e&t=db7247cb-300d-5b59-fd6c-6258fad20b89&l=127
> ERROR: ************* Module azext_aks_preview.azurecontainerstorage._validators
> src/aks-preview/azext_aks_preview/azurecontainerstorage/_validators.py:432:0: R0914: Too many local variables (27/25) (too-many-locals)

File: src/aks-preview/HISTORY.rst [hunk 1]
```diff
@@ -13,6 +13,11 @@ Pending
 +++++++
 * Update the minimum required cli core version to `2.61.0`.
 
+7.0.0b2
+++++++++
```
Comment (FumingZhang) [inline src/aks-preview/HISTORY.rst:17]:
```suggestion

7.0.0b2
++++++++
* Update the minimum required cli core version to `2.61.0`.
```

**Review by FumingZhang — APPROVED** [review]
LGTM

**Review by yanzhudd — APPROVED** [review]

**Pull request merged**